### ISSUE TITLE
Understanding your Workflow

### ISSUE THREAD

**Issue by dodgex** [androidannotations/androidannotations#1065]
Hello Guys,

I'm a big fan of AndroidAnnotations, I use it in my own Apps as well in the App we develop in the company where i work.

As I'm also adding PRs i would like to understand the Workflow of how you handle PR and new Versions to get the subjects of a PR to the users.

My [first PR](https://github.com/excilys/androidannotations/pull/736) was more or less a test to see if I can handle the way AA works while not really needing the changes for my project - this one got merged relatively fast (~12days). 
My [second one](https://github.com/excilys/androidannotations/pull/755) - with changes i actually needed - was first proposed on Oct. 6 2013 with an [update](https://github.com/excilys/androidannotations/pull/822) after merging the refactoring PR on Dec. 12 2013. The merge happened on Mar. 16 2014, about 160 days later.
The [current PR](https://github.com/excilys/androidannotations/pull/928)(actually [two](https://github.com/excilys/androidannotations/pull/929) seperate but related) was proposed on Mar. 2 2014 and is not yet(~120days later) merged - but it seems that a merge is in the near future.

While i understand that most of the work on AA is done in freetime, i would like to understand the politics behind when to merge a PR and when a new release is done. I have no problem with using a SNAPSHOT(i actually started with 3.0-SNAPSHOT and never had a stable :disappointed:) but since Oct 6. I'm forced to use a self built SNAPSHOT to have my changes available in my projects.

TL;DR
I would like to know how you decide when to merge a PR (after they are OK/LGTM) and when there will be a new release or at least a bump to the next minor snapshot (like 3.1.0-SNAPSHOT) where changes for the corresponding release will be merged as soon as they are OK/LGTM. For breaking changes (like a [plugin system](https://github.com/excilys/androidannotations/issues/957)) that would require a new Major i understand that they will open for a bit longer until there is enough to justify a jump to a new major release.

Thank you all for this beautiful library! ;)

@DayS @yDelouis
